theatre/studio/src/store/types/ahistoric.ts

```diff
@@ -1,8 +1,17 @@
 import type {ProjectState} from '@theatre/core/projects/store/storeTypes'
 import type {IRange, StrictRecord} from '@theatre/shared/utils/types'
+import type {Keyframe} from '@theatre/core/projects/store/types/SheetState_Historic'
+import type {PathToProp} from '@theatre/shared/utils/addresses'
+
+export type CopiedKeyframes = {
```

Comment:
Since we're including `pathToProp` and `trackId`, perhaps `CopiedKeyframes` is not descriptive enough of a name.

Reply:
Yeah I was thinking the same actually but was struggling to think of something. Will think some more...